### DISCUSSION TITLE
No mouse cursor

### DISCUSSION THREAD

**Discussion by TheFozid** [hyprwm/Hyprland#7588] [Q&A] [answered]
After months of trouble free use, my cursor has disappeared. Won't show over any window/tile. Can't find any errors anywhere. I'm on an AMD GPU if that helps? Not sure what to look at to find it? 
[code]Hyprctl setcursor[/code] returns ok when passing the relevant theme, but still no cursor.  Changing the gtk theme also doesn't show the cursor. 

I have [code]exec-once = hyprcursor[/code] in my config file, and it's all been working until today.

**Comment by MightyPlaza** [comment, marked as answer]
https://github.com/hyprwm/Hyprland/issues/7577

**Reply by TheFozid** [reply]
thanks 👍
